.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+**Behavior of the program**
+Please write a clear and concise description of what the bug is.
+
+**Expected behaviour**
+Please describe clearly and concisely what you expected to happen.
+
+**To Reproduce**
+Please add all steps to reproduce the behaviour:
+
+**Screenshots**
+If applicable, please add screenshots to help explain your problem. 
+
+**System Information**
+JRE Version: [Insert JRE Version here, a clear screenshot works too]
+
+/* To check JRE Version, open command prompt/terminal and type *java -version* */
+
+**Additional context**
+Add any other context about the problem here.
+
+
+Issue Template modified from FreeTube. 
+[https://github.com/FreeTubeApp/FreeTube]
```

Comment:
Bruh, I was just joking about this.

Reply:
bruh, too late. Well at least we wont get dmca or whatever like a certain other repo.

Reply:
The licence covers the code itself. Templates can be shared lmao

Reply:
so wht now

Reply:
It's not necessary.

Reply:
ok editing it now